gym/spec/package_command_generator_xcode7_spec.rb

```diff
@@ -15,7 +15,7 @@
 
       result = Gym::PackageCommandGeneratorXcode7.generate
       expect(result).to eq([
-                             "/usr/bin/xcrun #{Gym::XcodebuildFixes.wrap_xcodebuild} -exportArchive",
+                             "/usr/bin/xcrun #{Gym::XcodebuildFixes.wrap_xcodebuild.shellescape} -exportArchive",
```

Comment:
In `gym.rb`, please add `require 'shellwords'` to make sure the library gets imported